book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
getters and setters should also be present for the rentals since you'll be accessing and modifying it😉.
```suggestion
  attr_accessor :title, :author,  :rentals
```